assets/javascripts/initializers/disable-sort.js

```diff
@@ -0,0 +1,20 @@
+import discourseComputed from "discourse-common/utils/decorators";
+import { withPluginApi } from "discourse/lib/plugin-api";
+
+export default {
+  name: "disable-sort",
+  initialize() {
+    withPluginApi("0.8", (api) => {
+      api.modifyClass("component:topic-list", {
```

Comment:
@frank3manuel This `api.modifyClass()` is missing `pluginId` which results in warnings

```js
api.modifyClass("component:topic-list", {
  pluginId: "discourse-calendar",

  …
});
```

Reply:
Will fix this, thanks.

Reply:
Ha, too late! 😛 https://github.com/discourse/discourse-calendar/pull/325